lib/dentaku/evaluator.rb

```diff
@@ -162,5 +162,20 @@ def round_int(*args)
     def not(*args)
       Token.new(:logical, ! evaluate_token_stream(args[2..-2]).value)
     end
+
+    (Math.methods - Object.methods).each do |method|
+      define_method method do |*args|
```

Comment:
It's not a huge deal, but it seems like it might be better just to have a `delegate_to_math` (or some better name) method that all of these methods could call.